packages/openapi/README.md

```diff
@@ -15,3 +17,49 @@ From the monorepo root directory:
 ```shell
 pnpm --filter openapi test
 ```
+
+## Usage
+
+First, instantiate an `OpenAPI` validator object with a reference to your OpenAPI spec:
+
+```ts
+const openApi = await createOpenAPI(OPEN_API_URL)
```

Comment:
`createOpenAPI` accepts either a filepath or url of the OpenAPI spec 
https://github.com/APIDevTools/json-schema-ref-parser/blob/main/lib/index.d.ts#L35